backend/services/Auth/LoginHandler.js

```diff
@@ -0,0 +1,22 @@
+class LoginHandler {
+    constructor(userService, bcryptjs) {
+        this.userService = userService;
+        this.bcryptjs = bcryptjs;
+    }
+
+    async handle(email, password) {
+        const user = await this.userService.findOne({ email });
```

Comment:
I think we can add method to repository `findByEmail`

Reply:
done